src/fastapi_cli/cli.py

```diff
@@ -23,7 +24,12 @@
 
 def version_callback(value: bool) -> None:
     if value:
-        print(f"FastAPI CLI version: [green]{__version__}[/green]")
+        fastapi_version = version("fastapi")
+        python_version = platform.python_version()
+        fastapi_cli_version = version("fastapi-cli")
```

Comment:
This might be `fastapi-cli-slim`, and `fastapi` might be `fastapi-slim`. This might be why your tests are failing.

Reply:
so it's quite strange, because the rest of the tests are passing except this fastapi-slim

Reply:
That would be because `fastapi` is not installed. Instead, it's `fastapi-slim`. You'll probably want to keep using `__version__` and add `fastapi.__version__` like the original code.

Reply:
good idea friend  🫡

Reply:
@killjoy1221  Well, I did it this way using variables, do you have any suggestions?

Reply:
Don't use `importlib.metadata`.

Reply:
I think removing this won't make sense, just because of fastapi-slim

Reply:
@killjoy1221 Well, I solved the problem, it took a little work lol